ui/job-view/context/PinnedJobs.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { thPinboardCountError, thPinboardMaxSize } from '../../js/constants';
```

Comment:
What do you think about moving these two constants into this file?

Reply:
Ahh, good call...